ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -51,18 +57,36 @@ Licensed to the Apache Software Foundation (ASF) under one
   private final static Logger LOG = LoggerFactory.getLogger(LoadTable.class);
   //  private final Helper helper;
   private final Context context;
+  private final ReplLogger replLogger;
   private final TableContext tableContext;
   private final TaskTracker tracker;
   private final TableEvent event;
 
-  public LoadTable(TableEvent event, Context context, TableContext tableContext, TaskTracker limiter)
+  public LoadTable(TableEvent event, Context context, ReplLogger replLogger,
+                   TableContext tableContext, TaskTracker limiter)
       throws SemanticException, IOException {
     this.event = event;
     this.context = context;
+    this.replLogger = replLogger;
     this.tableContext = tableContext;
     this.tracker = new TaskTracker(limiter);
   }
 
+  private void createTableReplLogTask(String tableName, TableType tableType) throws SemanticException {
+    ReplStateLogWork replLogWork = new ReplStateLogWork(replLogger,tableName, tableType);
+    Task<ReplStateLogWork> replLogTask = TaskFactory.get(replLogWork, context.hiveConf);
+    ReplLoadTask.dependency(tracker.tasks(), replLogTask);
+
+    if (tracker.tasks().isEmpty()) {
```

Comment:
not sure why this would happen ? can be removed i think

Reply:
Just a defensive code for existingTableTasks flow which may not happen now. Will remove when we remove existingTableTasks later.